newroutes/contactUs.js

```diff
@@ -0,0 +1,71 @@
+const autoBind = require("auto-bind")
+const express = require("express")
+
+// Import middleware
+const { BadRequestError } = require("@errors/BadRequestError")
+
+const {
+  attachReadRouteHandlerWrapper,
+  attachWriteRouteHandlerWrapper,
+} = require("@middleware/routeHandler")
+
+const { UpdateContactUsSchema } = require("@validators/RequestSchema")
+
+class ContactUsRouter {
+  constructor({ contactUsPageService }) {
+    this.contactUsPageService = contactUsPageService
+    // We need to bind all methods because we don't invoke them from the class directly
+    autoBind(this)
+  }
+
+  // Read contactUs file
+  async readContactUs(req, res) {
```

Comment:
same comment as prev PR: should we prefer destructuring where possible?

Reply:
Yup, changed in https://github.com/isomerpages/isomercms-backend/pull/349/commits/d162665f610071571ef5c1eadbdd57fc78eadb50